migrations/v2.js

```diff
@@ -0,0 +1,464 @@
+
+  whereContent('adapt-contrib-slider - where slider', async content => {
+    sliders = getComponents('slider');
+    return sliders.length;
+  });
+
+  mutateContent('adapt-contrib-slider - add slider._canShowModelAnswer', async () => {
+    sliders.forEach(slider => {
+      slider._canShowModelAnswer = true;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-slider - check slider._canShowModelAnswer attribute', async () => {
+    const isValid = sliders.every(({ _canShowModelAnswer }) => _canShowModelAnswer === true);
+    if (!isValid) throw new Error('adapt-contrib-slider - _canShowModelAnswer not added to every instance of slider');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-slider - update to v2.0.4', { name: 'adapt-contrib-slider', version: '2.0.4', framework: '>=2.0.0' });
+
+  testSuccessWhere('non/configured slider component with empty course', {
+    fromPlugins: [{ name: 'adapt-contrib-slider', version: '2.0.3' }],
+    content: [
+      { _id: 'c-100', _component: 'slider' },
+      { _id: 'c-105', _component: 'slider' },
+      { _type: 'course' }
+    ]
+  });
+
+  testStopWhere('no slider components', {
+    fromPlugins: [{ name: 'adapt-contrib-slider', version: '2.0.3' }],
+    content: [{ _component: 'other' }]
+  });
+
+  testStopWhere('incorrect version', {
+    fromPlugins: [{ name: 'adapt-contrib-slider', version: '2.0.4' }]
+  });
+});
+
+describe('adapt-contrib-slider - v2.0.4 > v2.1.0', async () => {
+  let sliders;
+
+  whereFromPlugin('adapt-contrib-slider - from v2.0.4', { name: 'adapt-contrib-slider', version: '<2.1.0' });
+
+  whereContent('adapt-contrib-slider - where slider', async content => {
+    sliders = getComponents('slider');
+    return sliders.length;
+  });
+
+  mutateContent('adapt-contrib-slider - add slider._scaleStep', async () => {
+    sliders.forEach(slider => {
+      slider._scaleStep = 1;
+    });
+    return true;
+  });
+
+  mutateContent('adapt-contrib-slider - add slider._canShowModelAnswer', async () => {
+    sliders.forEach(slider => {
+      slider._canShowModelAnswer = true;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-slider - check slider._scaleStep attribute', async () => {
+    const isValid = sliders.every(({ _scaleStep }) => _scaleStep === 1);
+    if (!isValid) throw new Error('adapt-contrib-slider - _scaleStep not added to every instance of slider');
+    return true;
+  });
+
+  checkContent('adapt-contrib-slider - check slider._canShowModelAnswer attribute', async () => {
+    const isValid = sliders.every(({ _canShowModelAnswer }) => _canShowModelAnswer === true);
+    if (!isValid) throw new Error('adapt-contrib-slider - _canShowModelAnswer not added to every instance of slider');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-slider - update to v2.1.0', { name: 'adapt-contrib-slider', version: '2.1.0', framework: '>=2.0.0' });
+
+  testSuccessWhere('non/configured slider component with empty course', {
```

Comment:
The description here doesn't make sense. The migration script does not modify or rely on the course object.